release-notes.json

```diff
@@ -0,0 +1,40 @@
+{
+  "changes": [
+    {
+      "details": "Fixed Code that kills <b>Extension Host</b> for macOS",
+      "type": "FIX",
+      "color": "danger",
+      "author": "arnohovhannisyan",
+      "pullRequest": "834"
+    },
+    {
+      "details": "Download From <b>Public Gist</b> not working",
+      "type": "FIX",
+      "color": "danger",
+      "author": "shanalikhan",
+      "pullRequest": "816"
+    },
+    {
+      "details": "<b>Auto Upload</b> Fix",
+      "type": "FIX",
+      "color": "danger",
+      "author": "arnohovhannisyan",
+      "pullRequest": "835"
+    },
+    {
+      "details": "Inserts some <b>empty lines</b> in the beginning of settings.json",
+      "type": "NEW",
+      "color": "success",
+      "author": "knyhle",
+      "pullRequest": "828"
+    },
+    {
+      "details": "Update <b>Documentation</b>",
+      "type": "NEW",
+      "color": "success",
+      "author": "colinaaa",

```

Comment:
`author` and `pullRequest` supports empty string, right ?
Because for some issues I often do commit rather PR.

Reply:
I have just updated it to do so. You can either leave out those options or set them to empty strings to not show them.